index.js

```diff
@@ -89,12 +90,33 @@ function readConfig(cwd) {
   return config;
 }
 
+function readIntlConfig(cwd) {
+  let configPath = `${cwd}/config/ember-intl.js`;
+
+  let config;
+  if (fs.existsSync(configPath)) {
+    let requireESM = require('esm')(module);
```

Comment:
the default [ember-intl blueprint](https://github.com/ember-intl/ember-intl/blob/master/blueprints/ember-intl/files/__config__/ember-intl.js) is using `module.exports` (not `export default`), so I'm not sure why we would need to support ES module files here.

also, I would assume that the `default` export would still be a function, but it seems that the code below does not call the `default` and assumes that it is an object directly, right?

Reply:
Have `ember-intl` changed their setup recently because this PR worked when I opened it

Reply:
I'm not sure. I've checked in a few of my projects that use `ember-intl` and they all use `module.exports` 😅